src/components/application_manager/src/commands/mobile/add_sub_menu_request.cc

```diff
@@ -125,7 +124,10 @@ void AddSubMenuRequest::on_event(const event_engine::Event& event) {
             (*message_)[strings::msg_params][strings::menu_id].asInt(),
             (*message_)[strings::msg_params]);
       }
-      SendResponse(result, result_code, NULL, &(message[strings::msg_params]));
+      SendResponse(result,
+                   MessageHelper::HMIToMobileResult(result_code),
+                   response_info.empty()? NULL:response_info.c_str(),
```

Comment:
@dtrunov This check is not required. Also in the other places. Empty `std::string` call to `c_str()` returns `NULL`.